test/extended/util/oauthserver/oauthserver.go

```diff
@@ -59,10 +59,12 @@ var (
 	volumesDefaultMode int32 = 420
 )
 
+type NewRequestTokenOptionsFunc func(string, string) *tokencmd.RequestTokenOptions
+
```

Comment:
nit: could you please split the changes done to the util oauth server into a separate commit?

Reply:
Split into separate PR: https://github.com/openshift/origin/pull/23528